src/components/api-reference/schemas/measurements.js

```diff
@@ -390,6 +436,137 @@ class Measurements extends React.PureComponent {
           </Tabs>
         </div>
         <DataModelTable fields={STRATA_FIELDS} />
+
+        <h1 className='ds-h1' id='all-cause-readmission-measurements'>All Cause Readmission Measurements</h1>
+        <p className='ds-text--lead'>An All Cause Readmission Measurement represents beneficiaries aged 65 and older that were hospitalized at a short-stay acute-care hospital
```

Comment:
Could you add copy to indicate that third party integrators can't submit ACR or CAHPS measurements?